src/core/lib/event_engine/posix_engine/event_poller_posix_default.cc

```diff
@@ -31,14 +37,53 @@ namespace experimental {
 #ifdef GRPC_POSIX_SOCKET_TCP
 namespace {
 
+grpc_core::NoDestruct<std::vector<std::weak_ptr<ForkableInterface>>>
+    g_forkable_instances;
+
+bool g_registered{false};
```

Comment:
This can live inside the pthread guards.

Reply:
Done.